### ISSUE TITLE
Unclear Header Style

### ISSUE THREAD

**Issue by GanbaruTobi** [OWASP/ASVS#116]
In ASVS on Requirement 9.4 of the "Data protection verification requirements" it states the following:

Verify that the application sets appropriate anti-caching headers as per the risk of the application, such as the following: 
Expires: Tue, 03 Jul 2001 06:00:00 GMT
Last-Modified: {now} GMT
Cache-Control: no-store, no-cache, must-revalidate, max-age=0
Cache-Control: post-check=0, pre-check=0
Pragma: no-cache

This is confusing, since in [1] referred "There should only be one Cache-Control header (see rfc 2616) behaviour with multiple entries is undefined. Also the MSIE specific entries in the second cache-control are at best redundant."

[2] This Blog states the following: "If both post-check and pre-check are specified and set to 0, both are entirely ignored."

Should this part get a rework to be more clear?

[1] http://stackoverflow.com/questions/9884513/avoid-caching-of-the-http-responses
[2] https://blogs.msdn.microsoft.com/ieinternals/2009/07/20/internet-explorers-cache-control-extensions/

**Comment by GanbaruTobi** [comment]
To further push this:
https://twitter.com/ElijahLynn/status/685195442498473985

**Comment by vanderaj** [comment]
Resolved in v3.1

"Verify that the application sets sufficient anti-caching headers such that any sensitive and personal information displayed by the application or entered by the user should not be cached on disk by mainstream modern browsers (e.g. visit about:cache to review disk cache). "

I've not been specific as things change, but by including the acid test of what is acceptable, we get there in the end. I've also included some additional references to assist normal folks setting better headers.
